optaplanner-docs/src/main/asciidoc/index-docinfo.html

```diff
@@ -1,3 +1,10 @@
+<!-- Populate jstree for collapsed Table of Content-->
+<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/jstree/3.3.1/jstree.min.js"></script>
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/jstree/3.3.1/themes/default/style.min.css" />
+<link rel="stylesheet" href="jstree-style.css">
+<script src="toc.js"></script>
```

Comment:
Put these new links and scripts below the Google/Adabole analytics stuff (company policy). By putting the analytics first, we get a more accurate estimate on how many people open the docs but leave them very soon (for example because they load to slow - which we'd want to know)